src/test/java/org/folio/auth/authtokenmodule/AuthTokenTest.java

```diff
@@ -634,61 +662,237 @@ public void testSigningRequestGoodTokenBadPayload3() {
+        given()
+            .header("X-Okapi-Tenant", tenant)
+            .header("X-Okapi-Token", accessToken)
+            .header("X-Okapi-Url", "http://localhost:" + freePort)
+            .header("Content-type", "application/json")
+            .header("X-Okapi-Permissions", "[\"" + getMagicPermission("/refreshtoken") + "\"]")
+            .body(new JsonObject().put("userId", userUUID).put("sub", "jones").encode())
+            .post("/refreshtoken")
+            .then()
+            .statusCode(201)
+            .header("Content-Type", "application/json");
+      }
+
+      @Test
+      public void testSigningRequestUnsupportedMethod() {
+        given()
+            .header("X-Okapi-Tenant", tenant)
+            .header("X-Okapi-Token", accessToken)
+            .header("X-Okapi-Url", "http://localhost:" + freePort)
+            .header("Content-type", "application/json")
+            .header("X-Okapi-Permissions", "[\"" + getMagicPermission("/token/sign") + "\"]")
+            .body(new JsonObject().put("payload", payloadDummySigningReq).encode())
+            .put("/token/sign")
+            .then()
+            .statusCode(405);
+      }
+
+      @Test
+      public void testSigningRequestGoodTokenBadPayload() {
+        given()
+            .header("X-Okapi-Tenant", tenant)
+            .header("X-Okapi-Token", accessToken)
+            .header("X-Okapi-Url", "http://localhost:" + freePort)
+            .header("Content-type", "application/json")
+            .header("X-Okapi-Permissions", "[\"" + getMagicPermission("/token/sign") + "\"]")
+            .body("{")
+            .post("/token/sign")
+            .then()
+            .statusCode(400);
+      }
+
+      @Test
+      public void testSigningRequestGoodTokenBadPayload2() {
+        given()
+            .header("X-Okapi-Tenant", tenant)
+            .header("X-Okapi-Token", accessToken)
+            .header("X-Okapi-Url", "http://localhost:" + freePort)
+            .header("Content-type", "application/json")
+            .header("X-Okapi-Permissions", "[\"" + getMagicPermission("/token/sign") + "\"]")
+            .body(new JsonObject().put("noload", payloadDummySigningReq).encode())
+            .post("/token/sign")
+            .then()
+            .statusCode(400);
+      }
+
+      @Test
+      public void testSigningRequestGoodTokenBadPayload3() {
+        given()
+            .header("X-Okapi-Tenant", tenant)
+            .header("X-Okapi-Token", accessToken)
+            .header("X-Okapi-Url", "http://localhost:" + freePort)
+            .header("Content-type", "application/json")
+            .header("X-Okapi-Permissions", "[\"" + getMagicPermission("/token/sign") + "\"]")
+            .body(new JsonObject().put("payload", new JsonObject().put("x", 1)).encode())
+            .post("/token/sign")
+            .then()
+            .statusCode(400);
+      }
+
+      @Test
+      public void testRefreshTokenBadPayload3() {
```

Comment:
3?

Reply:
Not sure how that 3 crept in there. No need for it. Fixed